api/graphql/schema.graphql

```diff
@@ -28,38 +28,159 @@ enum Role {
 }
 
 type User {
-    # There are some cases where some fields, such as email and names, should be visible while others aren't. Therefore we use field-level permissions to distinguish them.
     id: ID!
-    email: Email!
     sub: String
+    roles: [Role]
+
+    # Personal info
     firstName: String @rename(attribute: "first_name")
     lastName: String @rename(attribute: "last_name")
+    email: Email!
     telephone: PhoneNumber
+    currentProvince: ProvinceOrTerritory @rename(attribute: "current_province")
+    currentCity: String @rename(attribute: "current_city")
+
+    # Language
+    languageAbility: LanguageAbility @rename(attribute: "language_ability")
     preferredLang: Language @rename(attribute: "preferred_lang")
-    roles: [Role]
+    lookingForEnglish: Boolean @rename(attribute: "looking_for_english")
+    lookingForFrench: Boolean @rename(attribute: "looking_for_french")
+    lookingForBilingual: Boolean @rename(attribute: "looking_for_bilingual")
+    bilingualEvaluation: BilingualEvaluation @rename(attribute: "bilingual_evaluation")
+    comprehensionLevel: EvaluatedLanguageAbility @rename(attribute: "comprehension_level")
+    writtenLevel: EvaluatedLanguageAbility @rename(attribute: "written_level")
+    verbalLevel: EvaluatedLanguageAbility @rename(attribute: "verbal_level")
+    estimatedLanguageAbility: EstimatedLanguageAbility @rename(attribute: "estimated_language_ability")
+
+    # Gov info
+    isGovEmployee: Boolean @rename(attribute: "is_gov_employee")
+    interestedInLaterOrSecondment: Boolean @rename(attribute: "interested_in_later_or_secondment")
+    currentClassification: Classification @belongsTo # Users current classification
+
+    # Employment equity
+    isWoman: Boolean @rename(attribute: "is_woman")
+    hasDisability: Boolean @rename(attribute: "has_disability")
+    isIndigenous: Boolean @rename(attribute: "is_indigenous")
+    isVisibleMinority: Boolean @rename(attribute: "is_visible_minority")
+
+    # Applicant info
+    jobLookingStatus: JobLookingStatus @rename(attribute: "job_looking_status")
     pools: [Pool] @hasMany # Pools that a user owns
     poolCandidates: [PoolCandidate] @hasMany # Pools candidates that a user owns
-    experiences: [Experience] # All experiences that a user owns
-    awardExperiences: [AwardExperience] # Award experiences that a user owns
-    communityExperiences: [CommunityExperience] # Community experiences that a user owns
-    educationExperiences: [EducationExperience] # Education experiences that a user owns
-    personalExperiences: [PersonalExperience] # Personal experiences that a user owns
-    workExperiences: [WorkExperience] # Work experiences that a user owns
+
+    hasDiploma: Boolean @rename(attribute: "has_diploma")
+    locationPreferences: [WorkRegion] @rename(attribute: "location_preferences")
+    locationExemptions: [String] @rename(attribute: "location_exemptions")
```

Comment:
locationExemptions was changed in the migration, but that change was missed here
```suggestion
    locationExemptions: String @rename(attribute: "location_exemptions")
```